ts/score/peer-score.ts

```diff
@@ -30,6 +30,14 @@ export class PeerScore {
    * IP colocation tracking; maps IP => set of peers.
    */
   peerIPs: Map<string, Set<string>>
+  /**
+   * Cache score up to decayInterval if topic stats are unchanged.
+   */
+  scoreCache: Map<string, number>
+  /**
+   * The time after which the cached score for a peer is no longer valid.
+   */
+  scoreCacheUntil: Map<string, number>
```

Comment:
Clarify exactly what number is: `unix timestamp in miliseconds`

Reply:
Consider merging `scoreCache` and `scoreCacheUntil`, it will reduce the memory cost and require a single set and prune operation